t/root_spamd.t

```diff
@@ -4,14 +4,14 @@
 
 use lib '.'; use lib 't';
 use SATest; sa_t_init("root_spamd");
-use Test;
 
-use constant TEST_ENABLED => conf_bool('run_root_tests');
-use constant IS_ROOT => eval { ($> == 0); };
-use constant RUN_TESTS => (TEST_ENABLED && IS_ROOT);
-
-BEGIN { plan tests => (RUN_TESTS ? 14 : 0) };
-exit unless RUN_TESTS;
+use Test::More;
+BEGIN {
```

Comment:
not sure the BEGIN block is required there
regarding the number of tests using this pattern would also consider a helper like skip_if_cannot_run_tests_as_root() to always perform these two checks